ecosystem/platform/server/app/controllers/application_controller.rb

```diff
@@ -17,14 +17,6 @@ def set_csrf_cookie
     }
   end
 
-  def after_sign_in_path_for(resource)
```

Comment:
This interfered with the `store_location_for` method. I think an alternative is to read the stored location here instead. Do we need the admin stuff right now? Especially in `config/routes.rb`:
```ruby
devise_for :users, **ActiveAdmin::Devise.config.deep_merge(...)
```
I think we'll need two separate `devise_for` calls there: one for users and one for admins.